### PR TITLE
feat: Add keys export command

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

**Review by omritoptix — APPROVED** [review]

File: cmd/keys/export/export.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	"github.com/spf13/cobra"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export <key-id>",
+		Short: "Exports the private key of the given key id",
```
Comment (mtsitrin) [inline cmd/keys/export/export.go:16]: what the expected key_id here? should be printed

File: cmd/keys/export/export.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+				utils.PrettifyErrorIfExists(err)
+				fmt.Println(out.String())
+			} else {
+				utils.PrettifyErrorIfExists(fmt.Errorf("invalid key id: %s. The supported keys are %s", keyID,
+					strings.Join(supportedKeys, ", ")))
+			}
+		},
+		Args: cobra.ExactArgs(1),
+	}
+	utils.AddGlobalFlags(cmd)
```
Comment (mtsitrin) [inline cmd/keys/export/export.go:45]: u can move it the root command

File: cmd/keys/export/export.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		Run: func(cmd *cobra.Command, args []string) {
+			home := cmd.Flag(utils.FlagNames.Home).Value.String()
+			config, err := utils.LoadConfigFromTOML(home)
+			utils.PrettifyErrorIfExists(err)
+			supportedKeys := []string{
+				consts.KeysIds.HubSequencer,
+				consts.KeysIds.RollappSequencer,
+			}
+			keyID := args[0]
+			if keyID == consts.KeysIds.HubSequencer {
```
Comment (mtsitrin) [inline cmd/keys/export/export.go:26]: add the options for "export all"

Comment (ItayLevyOfficial) [reply cmd/keys/export/export.go:26]: porque

File: cmd/keys/export/export.go (hunk 2 above)
Comment (ItayLevyOfficial) [reply cmd/keys/export/export.go:45]: Why do you ask? I find managing the keys via the keys subcommand to be quite convenient

File: cmd/keys/export/export.go (hunk 1 above)
Comment (ItayLevyOfficial) [reply cmd/keys/export/export.go]: fixed

File: cmd/keys/export/export.go (hunk 2 above)
Comment (mtsitrin) [reply cmd/keys/export/export.go:48]: I meant only the global flags

File: cmd/keys/export/export.go (hunk 3 above)
Comment (mtsitrin) [reply cmd/keys/export/export.go:29]: I think it'll be useful
don't u think?

Comment (ItayLevyOfficial) [reply cmd/keys/export/export.go:29]: unsure

File: cmd/keys/export/export.go (hunk 2 above)
Comment (ItayLevyOfficial) [reply cmd/keys/export/export.go]: good catch, fixed

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**